src/common/types/store-data/card-view-model.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { GuidanceLink } from '../../../scanner/rule-to-links-mappings';
+import { InstanceResultStatus, UnifiedResult } from './unified-data-interface';
```

Comment:
nitpick: some new lines in between this type/interface definitions would improve readability.